arcgis-ios-sdk-samples/Edit data/Create and save KML file/metadata.README.json

```diff
@@ -0,0 +1,32 @@
+{
+    "category": "Edit data",
+    "description": "Construct a KML document and save it as a KMZ file.",
+    "ignore": false,
+    "images": [
+        "create-save-kml-1.png",
+        "create-save-kml-2.png"
+    ],
+    "keywords": [
```

Comment:
I believe now we have `keywords` = `keywords` + `relevant_apis`.

Reply:
https://devtopia.esri.com/runtime/common-samples/wiki/README.metadata.json#keywords